src/runtime/server/middleware/session/index.ts

```diff
@@ -54,15 +87,19 @@ export const deleteSession = async (event: H3Event) => {
 }
 
 const newSession = async (event: H3Event) => {
-  // Cleanup old session data to avoid leaks
-  await deleteSession(event)
```

Comment:
Why was the cleanup removed?

Reply:
It was redundant in the flow and would be bug inducing with session-jacking attempts (cf. https://github.com/sidebase/nuxt-session/pull/16#discussion_r1016583833)